src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -44,6 +44,10 @@ export function textFormatter(row: number | undefined, cell: any | undefined, va
 	let valueToDisplay = '';
 	let titleValue = '';
 
+	if (value === '') {
```

Comment:
is this going to cause confusing for tables that contain empty string values?  i.e., will empty strings `''` incorrectly displayed as `NULL`?

Reply:
Upon first load, it does not appear to affect the table values and they do display properly, although when you press enter, it will change the value to an italicized null. (Is it expected for an empty string to be put in for nothing entered?)

Reply:
It is unnecessary to have it here, since empty strings are now entered in directly, so its been removed.